src/main/java/Event.java

```diff
@@ -0,0 +1,51 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class that inherits from Task.
+ * 
+ * @var from Representing start time
+ * @var to Representing end time
+ * 
+ * @author Owen Yeo
+ */
+public class Event extends Task{
+    private LocalDateTime from;
```

Comment:
Perhaps use a more descriptive name? eg fromTime? from feels abit too general it could be from (location) instead of from (time)?